physics/GFS_suite_interstitial.F90

```diff
@@ -683,6 +684,11 @@ subroutine GFS_suite_interstitial_4_run (im, levs, ltaerosol, cplchm, tracers_to
 
       real(kind=kind_phys), dimension(im,levs,ntrac), intent(inout) :: gq0
       real(kind=kind_phys), dimension(im,levs,nn),    intent(inout) :: clw
+      real(kind=kind_phys), dimension(im,levs),       intent(in) :: prsl
+      real(kind=kind_phys),                           intent(in) :: con_rd
+      real(kind=kind_phys), dimension(im,levs),       intent(in), optional :: nwfa, save_tcp
```

Comment:
The use of "optional" is a little different in CCPP. Need to remove it and change the metadata file as well.

Reply:
Ok, no problem